lib/active_merchant/billing/gateways/ebanx.rb

```diff
@@ -126,6 +128,14 @@ def verify(credit_card, options = {})
         end
       end
 
+      def inquire(authorization, options = {})
```

Comment:
I think `status` would be a better name for this method, since `inquire` is a term used very specifically upstream.

Reply:
Does this method fetch the status of a transaction? If so then I think keeping it `inquire` would be right

Reply:
I don't know, now that I think about it I think inquire is the way to go because it's basically doing the exact same thing as Transaction Sync. We're using the authorization to inquire about the payment status on Ebanx.

Reply:
@aenand yes it does

Reply:
Yeah I'd agree that we should use `inquire` for any method that takes in a transaction reference, fetches the status from the gateway, and returns it